.gitignore

```diff
@@ -3,12 +3,19 @@
 .bundle
 build/*
 tools/chef/cookbooks/*
+tools/chef/site-cookbooks/**/cookbooks/*
```

Comment:
The less the developer has to think about excluding, the more cookbooks they may create, rather than lumping everything in project``